### PR TITLE
[irods/irods#6597] Add Ubuntu 22.04 support

### PR DESCRIPTION
**Pull request by alanking**
I don't remember if this works all the way, but putting up for interested eyeballs :)

---

Companion PRs:
https://github.com/irods/irods/pull/6620
https://github.com/irods/irods_client_icommands/pull/323
https://github.com/irods/irods_development_environment/pull/72

### CODE REVIEW

File: projects/ubuntu-22.04/Dockerfile [hunk 1]
```diff
@@ -1,36 +1,39 @@
...
+        apt-transport-https \
+        gnupg \
+        wget \
+    && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/* /tmp/*
+
 #RUN wget -qO - https://packages.irods.org/irods-signing-key.asc | apt-key add - && \
-    #echo "deb [arch=amd64] https://packages.irods.org/apt/ jammy main" | tee /etc/apt/sources.list.d/renci-irods.list
+    #echo "deb [arch=amd64] https://packages.irods.org/apt/ focal main" | tee /etc/apt/sources.list.d/renci-irods.list
```
Comment (alanking) [inline projects/ubuntu-22.04/Dockerfile:15]: Oops

These will need to be uncommented in the future when we have published externals packages, anyway

Comment (korydraughn) [reply projects/ubuntu-22.04/Dockerfile:15]: And this is just a note to yourself to do that once everything aligns, correct?

Comment (alanking) [reply projects/ubuntu-22.04/Dockerfile:15]: Yeah, once we have packages for jammy these will need to be uncommented. Also I accidentally changed the jammy to a focal here. :/

**Comment by alanking** [comment]
Unit tests and core tests passed. Also confirmed that catalog consumer instances are working

File: README.md [hunk 2]
```diff
@@ -28,6 +30,31 @@ pip freeze
...
+
+Check out the latest tag which was still using the Python implementation:
+```bash
+cd compose
+git checkout 1.29.2
+```
+
+At this point, you can make the modifications needed to fix any problems you may encounter.
+
+One done, back out and pip install the local directory:
```
Comment (korydraughn) [inline README.md:52]: One -> Once?

**Review by korydraughn — COMMENTED** [review]
Excellent! Once the comment is resolved, squash and we'll approve.

**Review by korydraughn — APPROVED** [review]
Add the pound when ready.

**Pull request merged**